content/W2015/W15-event-confidence.md

```diff
@@ -0,0 +1,25 @@
+Title: Big CSters Discussion Circle: Self-Confidence
+Date: 2015-03-15 20:00
+Category: Events
+Tags: social, big csters
+Slug: csters-confidence
+Author: Julia Nguyen
+Summary: Big CSters convenes for a discussion circle around the topics of imposter syndrome and self-confidence.
```

Comment:
Nope. The summary line has to be all on one line, or else the software won't parse it correctly. It's the only exception to the 80-char lines rule :)